CenterForOpenScience/osf.io
# Email changes for registries moderation [ENG-2439]

## Purpose

There needs to be some email changes for registries moderation

## Changes

Modifying website templates for emails. Updating business logic

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify Emails are correct
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A

## Side Effects

N/A
## Ticket

https://openscience.atlassian.net/browse/ENG-2439
